docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.8'
+
+services:
+  web:
+    build: .
+    ports:
+      - "3000:3000"
+    environment:
+      DATABASE_URL: postgresql://citadel:citadel@postgres:5432/citadel
+      RAILS_ENV: production
+      RAILS_LOG_TO_STDOUT: "true"
+    volumes:
+      - ./config/environments/production.override.rb:/citadel/config/environments/production.override.rb:ro
+      - ./config/credentials.yml.enc:/citadel/config/credentials.yml.enc:ro
+      - ./config/master.key:/citadel/config/master.key:ro
+      - ./public/uploads:/citadel/public/uploads
+    depends_on:
+      postgres:
+        condition: service_healthy
+    restart: unless-stopped
+
+  postgres:
+    image: postgres:16-alpine
+    environment:
+      POSTGRES_USER: citadel
+      POSTGRES_PASSWORD: citadel
+      POSTGRES_DB: citadel
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
```

Comment:
If your firewall wasn't properly set up this would expose the database to the internet with a very weak password, an incredibly insecure default.

Reply:
Good point ! I removed this, since the Web service is talking directly to the database through the docker network.